### PR TITLE
List Paxos Gold (PAXG.grv)

### PR DESCRIPTION
**Pull request by shabbydev**
## Description

Adding token: PAXG
from chain: Gravity Bridge
Pool: PAXG.grv/USDC, 1492
Pool: https://celatone.osmosis.zone/osmosis-1/pools/1492

### Adding Assets

If adding a new asset, please ensure the following:
- [ ] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the representation and transferring of the new asset has been validated)

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).): (choose one)
- [x] Ready -- A liquidity pool(s) has been created and has liquidity. The pool ID is: 1492
- [ ] Soon -- A pool will be created soon. Do not merge this PR until liquidity is ready. 
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Note: This method is not recommended because the Supercharged Liquidity pool type cannot be created via the Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds. The StreamSwap Bootstrapping phase begins (when?): ______

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -716,6 +716,27 @@
...
+    {
+      "chain_name": "gravitybridge",
+      "base_denom": "gravity0x45804880De22913dAFE09f4980848ECE6EcbAf78",
+      "path": "transfer/channel-144/gravity0x45804880De22913dAFE09f4980848ECE6EcbAf78",
+      "osmosis_verified": false,
+      "override_properties": {
+        "name": "Paxos Gold (Gravity Bridge)",
+        "symbol": "PAXG.grv",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/paxg.grv.svg"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:728]: This file doesn't exist. If you can register PAXG to /_non-cosmos/ethereum/, I can create it in the same style.

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -3391,6 +3391,65 @@
         "peg:collateralized"
       ]
     },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:3394]: Please do not add to the assetlist.json file directly. This is generated only

Comment (shabbydev) [reply osmosis-1/osmosis-1.assetlist.json]: thank you, removed

File: osmosis-1/osmosis.zone_assets.json (hunk 1 above)
Comment (shabbydev) [reply osmosis-1/osmosis.zone_assets.json:728]: added paxg.grv.svg in same style of other assets in https://github.com/cosmos/chain-registry/commit/40c02237f8803fd65e0f1f145d6093b920e74937

**Pull request merged**